publications-2014.bib

```diff
@@ -1175,7 +1175,7 @@ @article{2014_109
 }
 
 @article{2014_110,
-   author  = {A. Vidal-Ferrandiza and R. Fayeza and D. Ginestarb and G. Verdúa},
+   author  = {A. Vidal-Ferrandiza and R. Fayeza and D. Ginestarb and G. Verd\'{u}a},
```

Comment:
apparently, same `Verdu`, see https://www.sciencedirect.com/science/article/pii/S0306454914002473

Reply:
oh, all surnames have extra `a`/`b` here

Reply:
Indeed :-)

Reply:
I bet they were copied verbatim from a list of authors where a footnote symbol referred to an address at the bottom of the page. Might have been my fault.

Reply:
Yip, I agree that this is probably what happened. Although it wasn't a big deal to fix the issue now, I don't think that we should worry too much about this right now. We'll have to go through all of the entries to double check the data anyway.